### ISSUE TITLE
Implement HmIPW-DRD3

### ISSUE THREAD

**Issue by hobbyquaker** [rdmtc/RedMatic-HomeKit#241]

**Comment by hobbyquaker** [comment]
Bisher ungetestet

**Comment by stefanfejfar** [comment]
Funktioniert tadellos. Allerdings werden die Geräte als SingleAccessory in Homekit ausgegeben. Ebenso kann ich die drei Kanäle nicht unterschiedlichen Räumen zuweisen, sondern alle drei liegen in einem.

**Comment by c0d35io** [comment]
> Ebenso kann ich die drei Kanäle nicht unterschiedlichen Räumen zuweisen, sondern alle drei liegen in einem.

Genau, das macht in meinem Fall auch keinen Sinn bzw. kann ich deshalb die Homekit-Erweiterung nicht nutzen, da an einem DRD3 teilweise Lampen aus verschiedenen Räumen hängen.

**Comment by HemanMaster** [comment]
Danke erstmal, das man die Kanäle unterschiedlichen Räumen zuweist ist immer noch so oder? Gleiche Thema hab ich mit den HmIPW-DRS8. Beim DRI32 z.B. funktioniert das.

**Comment by sreinhold95** [comment]
Als SingleAccessory funktioniert er sehr gut. Nicht als SingleAccessory wäre dann perfekt.
als Tester  für nicht SingleAccessory  wäre kein Problem von meiner Seite.

**Comment by sreinhold95** [comment]
Ich habe einen Fix für den DRD3 geschrieben. er geht jetzt per Single oder MultiAcessory
[redmatic-homekit-homematic-devices.zip](https://github.com/rdmtc/RedMatic-HomeKit/files/8844776/redmatic-homekit-homematic-devices.zip)
. #353

**Comment by Flobo2012** [comment]
Guten Tag zusammen, 
kann mir bitte jemand zeigen wo ich eine Anleitung im Forum finde, wie ich die Datei des DRD3 "update"? 
Ich komme trotz Recherchen im Netz und Forum einfach nicht weiter.

Danke für die Hilfe

**Comment by NullcheckerHipw** [comment]
Hallo zusammen, 
ich habe das gleiche Problem und ich bin da leider nicht so fit in dem Thema. Kann das evtl. jemand erklären, wie man das jetzt updatet? 
Oder zumindest einen Link zur Anleitung teilen, wäre schon hilfreich. 

Danke vorab.

**Comment by hobbyquaker** [comment]
RedMatic-HomeKit **4.0.0** ist seit heute veröffentlicht – Installation in RedMatic 9 über den Node-RED-Palettenmanager (Suche nach `redmatic-homekit`), Release-Notes: https://github.com/rdmtc/RedMatic-HomeKit/releases/tag/v4.0.0.

Der HmIPW-DRD3 hat ein eigenes Modul; in 4.0.0 lassen sich die drei Ausgänge außerdem auf einzelne Accessories aufteilen (Option *SingleAccessory* abwählen, siehe #360).

Falls es mit 4.0.0 nicht wie erwartet funktioniert: bitte hier melden (Gerätetyp, Node-RED-Log mit Level „debug“ rund um den Start), dann wird das Issue wieder geöffnet.

---
*Written by Claude Fable on behalf of hobbyquaker.*
